src/ui/common/src/components/integrations/buttonGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import { faFileCsv, faFlask } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Tooltip from '@mui/material/Tooltip';
+import React from 'react';
+
+import { Integration } from '../../utils/integrations';
+import ButtonGroup from '../primitives/ButtonGroup';
+import { IconButton } from '../primitives/IconButton.styles';
+
+type Props = {
+  integration: Integration;
+  onUploadCsv?: () => void;
+  onTestConnection?: () => void;
+};
+
+const IntegrationButtonGroup: React.FC<Props> = ({
+  integration,
+  onTestConnection,
+  onUploadCsv,
+}) => {
+  return (
+    <ButtonGroup>
+      <Tooltip title="test-connect this integration">
```

Comment:
I think for the tooltips, you should capitalize the first word.
For this tooltip, I think it would be more clear to say "Test integration connection"

Reply:
Thanks! We actually removed these tooltips in #293 so they won't be an issue now!